From a4ec3f654a4be50aaf268c3a540e8b110ad2ded3 Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Tue, 6 Nov 2018 18:59:47 +0100
Subject: [PATCH] debian: Enable SMP in userspace package

Hard to find a UP target these days, and we definitely want to be
generic with Debian packages.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
---
 debian/rules | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 99ed8982ce..883ae3ed8c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -16,7 +16,8 @@ DEB_BUILD_GNU_TYPE=$(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 CONFIG_OPTS = --prefix=/usr \
             --includedir=/usr/include/xenomai \
             --mandir=/usr/share/man \
-	    --with-testdir=/usr/lib/xenomai/testsuite
+            --with-testdir=/usr/lib/xenomai/testsuite \
+            --enable-smp
 
 ifneq ($(DEB_HOST_GNU_TYPE),$(DEB_BUILD_GNU_TYPE))
 	CONFIG_OPTS += --build $(DEB_BUILD_GNU_TYPE) --host $(DEB_HOST_GNU_TYPE)
-- 
2.16.4

